interfaces/builtin/docker_support.go

```diff
@@ -629,6 +636,11 @@ func (iface *dockerSupportInterface) BeforePreparePlug(plug *snap.PlugInfo) erro
 			return fmt.Errorf("docker-support plug requires bool with 'privileged-containers'")
 		}
 	}
+	if v, ok := plug.Attrs["controls-device-cgroup"]; ok {
+		if _, ok = v.(bool); !ok {
+			return fmt.Errorf("docker-support plug requires bool with 'controls-device-cgroup'")
```

Comment:
I think the message is a little confusing, perhaps more so to reviewers than to users. It seems to say that a plug must always have the attribute named `controls-device-cgroup`. I think the desired message is that if present, the attribute must be of type bool.

This is not a blocker but if I'm reading this right, please adjust this in a follow up.

Reply:
actually this is no longer an attribute as per @jdstrand, so this whole block shouldn't be here, my apologies